seed/views/v3/columns.py

```diff
@@ -96,7 +98,7 @@ def list(self, request):
         """
         organization_id = self.get_organization(self.request)
         inventory_type = request.query_params.get('inventory_type', 'property')
-        only_used = request.query_params.get('only_used', False)
+        only_used = json.loads(request.query_params.get('only_used', 'false'))
```

Comment:
Note that we were previously getting the query param as a string, which means when doing conditionals with it later it would always be true whenever this param was set.